src/core/recv_buffer.c

```diff
@@ -154,6 +154,9 @@ QuicRecvBufferHasUnreadData(
     _In_ QUIC_RECV_BUFFER* RecvBuffer
     )
 {
+    if (RecvBuffer->ReadPendingLength != 0 && RecvBuffer->RecvMode != QUIC_RECV_BUF_MODE_MULTIPLE) {
+        return FALSE;
+    }
```

Comment:
Checks ReadPendingLength instead of app to manage flush api.
This seems to be indicating FIN flag to the stream callback. Is this ok? No test issue.